py_gnome/gnome/maps/map.py

```diff
@@ -389,9 +394,133 @@ def resurface_airborne_elements(self, spill_container):
         np.maximum(next_positions[:, 2], 0.0, out=next_positions[:, 2])
         return None
 
-    def to_geojson(self):
-        return FeatureCollection([])
-
+    def gnm_poly_2_ogr_poly(self, gnm_points):
+        ring = ogr.Geometry(ogr.wkbLinearRing)
+        for point in gnm_points:
+            ring.AddPoint(point[0], point[1])
+        if (gnm_points[0][0] != gnm_points[-1][0]) or (gnm_points[0][1] != gnm_points[-1][1]):
+            first_point = gnm_points[0]
+            ring.AddPoint(first_point[0], first_point[1])
+        ogr_poly = ogr.Geometry(ogr.wkbPolygon)
+        ogr_poly.AddGeometry(ring)
```

Comment:
wow! this is pretty painful :-( 

is it possible to do this more directly by passing geoJSON-compatible Python in?

Or we should put utilities like this in a separate geometry sub package (the one in gnome,utilities needs some (a lot) of refactoring, but that's the idea)